src/vm/assembly.cpp

```diff
@@ -2588,6 +2588,7 @@ HRESULT RunMain(MethodDesc *pFD ,
             // Set the return value to 0 instead of returning random junk
             *pParam->piRetVal = 0;
             threadStart.Call(&stackVar);
+            *pParam->piRetVal = GetLatchedExitCode();
```

Comment:
@jkotas @RussKeldorph I checked the test itself, and found that it is out of synch as aforementioned. CoreCLR from Linux.x64.Debug build also fails for the old test.

Is there any plan to support System.Environment.ExitCode property? 

It would be very helpful if there is any document that shows which methods/properties will be supported in CoreCLR when investigating issues of this kind.